compiler/sem/semstmts.nim

```diff
@@ -484,6 +485,13 @@ proc newSymChoiceUseQualifierReport(n: PNode): SemReport =
   for child in n:
     result.symbols.add child.sym
 
+proc newSymChoiceUseQualifierDiag(n: PNode): PAstDiag =
+  assert n.kind in nkSymChoices
+  result = PAstDiag(kind: adSemAmbiguousIdent, 
+                    wrongNode: n)
+  for child in n:
+    result.ambSyms.add child.sym
```

Comment:
Would it make sense to leave querying the symbols to the display/rendering logic?

Reply:
aaaaaaaaand done! 🎉 

Thanks for spotting that.